.github/workflows/pullrequest.yml

```diff
@@ -14,8 +14,6 @@ jobs:
         ports:
           - 8013:8013
 
-    permissions:
-      packages: write
```

Comment:
Do we need this [here](https://github.com/open-feature/java-sdk/blob/73cc784491f6dc0a371a2bc44d780fa5e3d39eaa/.github/workflows/merge.yml#L21-L22)?

Reply:
I don't think so. Added https://github.com/open-feature/java-sdk/pull/130